com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added support for importing the cage mesh of SubD Alembic nodes.
 
 ### Changed
+- Unsupported schema nodes are imported as NOOP GameObjects.
```

Comment:
Alembic importer now imports unsupported schema nodes as empty GameObjects.

Reply:
no-op maybe? NOOP reads like a type of game objects